ir/src/transition_stmts/graph.rs

```diff
@@ -287,6 +286,16 @@ impl AlgebraicGraph {
                     )))
                 }
             }
+            IdentifierType::TraceColumns(columns) => {
+                let trace_segment = columns.trace_segment();
+                let col_idx = columns.offset() + vector_access.idx();
+                let node_index = self.insert_op(Operation::TraceElement(TraceAccess::new(
+                    trace_segment,
+                    col_idx,
+                    0,
+                )));
```

Comment:
nit: Maybe we can use constants `CURRENT_ROW` and `NEXT_ROW` for the row offsets of 0 and 1 or at least add comments where these `TraceAccess` structs are created? I think it would be easier to read what's going on for anyone who is new to the codebase or has stepped away for a week and apparently forgotten everything ;)

Reply:
Changed it to use `CURRENT_ROW` and `NEXT_ROW`. Thanks for the suggestion, I used to get confused with this one as well.